cLIMS/organization/templates/detailConstructs.html

```diff
@@ -7,9 +7,9 @@
  <script src="{% static "siteWide/js/show.js" %}"></script> 
 {% if construct %}
 <div class="bodyContent container">
-<a href="#">
+<!--<a href="#">
```

Comment:
i wonder what did this button even do before ? https://stackoverflow.com/questions/4855168/what-is-href-and-why-is-it-used

`href=#` goes scrolls on top of the pcurrent page without doing much... am I missing something ?

Reply:
Yes it just goes to the top of the page.
So when you click on edit/delete button.
If you check the url , # is appended, and stays on the same page